doc/classes/ConcavePolygonShape.xml

```diff
@@ -5,6 +5,7 @@
 	</brief_description>
 	<description>
 		Concave polygon shape resource, which can be set into a [PhysicsBody] or area. This shape is created by feeding a list of triangles.
+		Note: when used for collision, [ConcavePolygonShape] is intended to work with static [PhysicsBody] and will not work with [KinematicBody] or [RigidBody] with a mode other than Static.
```

Comment:
...intended to work with static [PhysicsBody] **nodes** and...

May also be worth mentioning [StaticBody] somewhere.

Reply:
I made the changes.